examples/lusid/fx-forwards/create-fx-forward.sql

```diff
@@ -0,0 +1,30 @@
+-- ===============================================================
+-- Description:
```

Comment:
Could the description match that of the upload equity example with the bullet point breakdown? It is a bit overkill but makes it very clear what is happening.

Reply:
Formatted matched